pkgs/development/tools/build-managers/dub/default.nix

```diff
@@ -51,9 +51,12 @@ let
 
   # Need to test in a fixed-output derivation, otherwise the
   # network tests would fail if sandbox mode is enabled.
-  # Disable tests on Darwin for now because they don't work
-  # reliably there.
-  dubUnittests = if !stdenv.hostPlatform.isDarwin then
+  #
+  # Tests are disabled by default because they are impure.
+  # They rely on state from servers on the Internet and though
+  # should only be run manually once after changes to the package
+  # were made.
+  dubUnittests = if false then
```

Comment:
Could we instead disable just those network tests? That would allow to remove fixed-output hack and prevent test results from being cached.